torch/_dynamo/variables/builtin.py

```diff
@@ -340,6 +373,15 @@ def call_function(
             )
         return super().call_function(tx, args, kwargs)
 
+    def _call_reversible(self, tx, forward_name, reverse_name, args, kwargs):
+        # Call reverse (e.g. __radd__) if forward (e.g. __add__) isn't supported.
+        # This can be the case when doing int + SymInt, for example.
+        forward_res = args[0].call_method(tx, forward_name, args[1:], kwargs)
+        if forward_res is NotImplemented:
```

Comment:
This is a little suspect, because the method should return a variable tracker, not a "real" NotImplemented

Reply:
You should return a ConstantVariable of NotImplemented. You also need to test if the rhs is NotImplemented and raise an error (probably just unimplemented?) if so, since both sides missing induces TypeError (but Dynamo's general handling of error cases is to bail)